packages/contracts-bedrock/test/vendor/eas/EAS.t.sol

```diff
@@ -0,0 +1,2758 @@
+                    block.chainid,
+                    address(eas)
+                )
+            );
+    }
+
+    // Helper function to create request data
+    function createAttestationRequestData()
+        internal
+        view
+        returns (AttestationRequestData memory)
+    {
+        return
+            AttestationRequestData({
+                recipient: recipient,
+                expirationTime: uint64(block.timestamp + 30 days),
+                revocable: true,
+                refUID: ZERO_BYTES32,
+                data: hex"1234",
+                value: 0
+            });
+    }
+
+    // Helper function to create attestation digest
+    function _createAttestationDigest(
+        bytes32 schemaId,
+        AttestationRequestData memory data,
+        address attester,
+        uint64 deadline
+    ) internal view returns (bytes32) {
+        // Match the exact order from EIP1271Verifier.t.sol
+        bytes32 structHash = keccak256(
+            abi.encode(
+                getDelegatedAttestationTypeHash(),
+                attester, // attester first
+                schemaId, // then schema
+                data.recipient,
+                data.expirationTime,
+                data.revocable,
+                data.refUID,
+                keccak256(data.data),
+                data.value,
+                0, // nonce
+                deadline
+            )
+        );
+
+        return
+            keccak256(
+                abi.encodePacked(
+                    "\x19\x01",
+                    createDomainSeparator(),
+                    structHash
+                )
+            );
+    }
+
+    // Helper function for revocation digests
+    function _createRevocationDigest(
+        bytes32 schemaId,
+        bytes32 uid,
+        address revoker,
+        uint64 deadline,
+        uint256 nonce
+    ) internal view returns (bytes32) {
+        bytes32 structHash = keccak256(
+            abi.encode(
+                REVOKE_TYPEHASH,
+                revoker,
+                schemaId,
+                uid,
+                0,
+                nonce,
+                deadline
+            )
+        );
+
+        return
+            keccak256(
+                abi.encodePacked(
+                    "\x19\x01",
+                    createDomainSeparator(),
+                    structHash
+                )
+            );
+    }
+
+    // =============================================================
+    //                     SETUP
+    // =============================================================
+    function setUp() public {
+        // Initialize senderKey
+        senderKey = 0x12345;
+        sender = vm.addr(senderKey);
+
+        // Setup accounts
+        sender2 = makeAddr("sender2");
+        recipient = makeAddr("recipient");
+        recipient2 = makeAddr("recipient2");
+
+        // Create registry at a temporary address
+        registry = new SchemaRegistry();
```

Comment:
If we use `CommonTest` like noted in the other comment, we don't need to deploy the `registry` here